packages/ui/src/Customization/ExtensionOverview.css

```diff
@@ -0,0 +1,8 @@
+.property-label {
```

Comment:
namespace should be the same as the filename

Reply:
Will do